### ISSUE TITLE
Ingress Controller

### ISSUE THREAD

**Issue by btat** [SUSE/doc-cap#357]
https://trello.com/c/jgev7lC4/892-3-fissile-work-to-use-an-ingress-controller

**Comment by cjschroder** [comment]
ok another one not ready for docs-- we must have the doc coordination meeting and talk about workflow. It's no good opening Issues for unready features that are not documented.

**Comment by btat** [comment]
This (and the other similar not quite ready for docs issues opened) is probably me jumping the gun a bit on creating the issues. I thought it'd be nice to have on our radar/tracking board of upcoming items tagged as needing docs attention from the sprint board. I agree though, we should discuss this during the meeting. Will you be joining this week's meeting since you're at the offsite our save this for the following week?

**Comment by cjschroder** [comment]
Ok that makes sense, let's discuss in a meeting.

**Comment by gaktive** [comment]
Wiki page: https://github.com/SUSE/scf/wiki/Using-Ingress-Controller-with-SCF

**Comment by troytop** [comment]
Note: We are still trying to decide on a sensible default for the `nginx.ingress.kubernetes.io/proxy-body-size` (it's currently set much to low for `cf push` to work with large files), but we should document how to increase this value if needed (e.g. `kubectl edit ingress --namespace scf scf`). 

This is the error you will see if `proxy-body-size` is set to low for the app you are pushing:

```
omparing local files to remote cache...
Packaging files to upload...
Uploading files...
 0 B / 31.29 MiB [----------------------------------------------------------------------------------------------------]   0.00%Error unmarshalling the following into a cloud controller error: <html>
<head><title>413 Request Entity Too Large</title></head>
<body bgcolor="white">
<center><h1>413 Request Entity Too Large</h1></center>
<hr><center>nginx/1.15.3</center>
</body>
</html>

FAILED
```

**Comment by troytop** [comment]
@satadruroy or @f0rmiga can provide instructions for changing this setting.

**Comment by f0rmiga** [comment]
It can be passed through the `ingress.annotations`:

```yaml
ingress:
  enabled: true
  annotations:
    nginx.ingress.kubernetes.io/proxy-body-size: 1024m
```

Ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md#custom-max-body-size

**Comment by btat** [comment]
Resolved by #419
